### PR TITLE
contao 5 compatibility

### PR DESCRIPTION
**Pull request by kampfq**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   ],
   "require": {
     "php": ">=7.4",
-    "contao/core-bundle": "~4.9"
+    "contao/core-bundle": "~4.9 || ^5.0"
```
Comment (rabauss) [inline composer.json]: Maybe we should drop the support for 4.9
```suggestion
    "contao/core-bundle": "^4.13 || ^5.0"
```

Comment (denniserdmann) [reply composer.json]: You're right. I guess some changes are not supported in Contao 4.9

Comment (kampfq) [reply composer.json]: yes, sorry for that. i forgot to raise the minimum version.

**Comment by denniserdmann** [comment]
@kampfq Thanks for your PR, although we couldn't merge it (as some errors appeared, when i tested it). 

But by now, the new Version 5.0 is finally compatible with Contao 5

**Pull request closed without merging**